index.html

```diff
@@ -42,7 +42,159 @@ <h1>I'm Denzel Glad to see you!</h1>
                 <span><a href="#"><i class="fab fa-angellist"></i></a></span>
             </div>
         </section>
+        <section class="projects_section">
+            <div class="project">
+                <div class="image_div">
+                    <img src="/images/Snapshoot-Portfolio.png" alt="project image" class="project_image">
+                </div>
+                <div class="details_div">
+                    <h2 class="project_title">Tonic</h2>
+                    <ul class="some_details">
+                        <li class="bold">CANPOY</li>
+                        <li class="tech"><i class="fas fa-circle"></i>Back End Dev</li>
+                        <li class="tech"><i class="fas fa-circle"></i>2015</li>
+                    </ul>
+                    <p class="project_details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="technologies_used">
 
+                        <img src="/images/Tag-1.png" alt="html-projects">
+                        <img src="/images/Tag.png" alt="css-projects">
+                        <img src="/images/Tag-2.png" alt="javaScritp-projects">
```

Comment:
For all of the images that you use in your project you should add a point "." before first "/" in src attribute to pass from:
`<img src="/images/Tag-1.png" alt="html-projects">`
to:
`<img src="./images/Tag-1.png" alt="html-projects">`
For this reason, I can't see the images in the images you used, please add this point "." before the first "/" in your src attribute for all your images.

- [ ] Fix images in your HTML file.